generator-tests/infinity-g1.txt

```diff
@@ -1,7 +1,7 @@
 ff01ffffffa00101010101010101010101010101010101010101010101010101010101010101ffff01ffff2bffb0c00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000ff86666f6f6261728080ff7bff80808080
 SPENDS:
 - coin id: 10d6c244183579e7704ea71e862098f15d28d63fb63c6a54779d1fe901ca48b1 ph: 99d9c5fd60cc6d5bca3390783d3489cd744139f7f143191ea911de3a22b50907
-cost: 1352512
+cost: 2552512
```

Comment:
this generator uses an `AGG_SIG_PARENT` conditions (43 / `0x2b`) which doesn't incur any cost unless the conditions are enabled. Enabling them now causes this test to cost more